src/flanders/markdown.cljc

```diff
@@ -37,8 +37,9 @@
 (defleaf ReferenceNode [text anchor jump-anchor])
 
 (defn ready-for-table [str]
-  (str/replace str #"(\n|\|)" {"\n" " "
-                               "\\|" "\\\\|"}))
+  (-> str
+      (str/replace \newline \space)
+      (str/replace "|" "\\\\|")))
```

Comment:
by changing this function I believe you are passing twice on the full string instead of taking advantage of making a single pass.

Also, it looks like the logic is changed, not sure why we previously replaced `\|` by `\\|` instead of `|` by `\\|` but this might break something elsewhere.